### PR TITLE
Robot skeleton wings

### PR DESCRIPTION
**Pull request by Mqiib**
# General Documentation

### Intent of your Pull Request

Adds wings for both preternises (preterni?) and skeletons, sprited by yours truly. Skeletons still can't fly.

image
image

### Why is this change good for the game?

More wings = good, skeletons still not op

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Skeletons and preternisissesises can have wings.

### What should players be aware of when it comes to the changes your PR is implementing?

To open/close wings for skeletons you have to use the *wing emote because toggle flight won't work (on account of them not being able to fly).

### What general grouping does this PR fall under? 

Spriting

# Changelog

:cl:  
rscadd: Skeletons and preternisnseresiisnrisses can now get wings from the wing potion! Skeletons cannot fly, but the robots can.
imageadd: Skeleton and robotic wing sprites added!
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
Merge remote-tracking branch 'MqiibUpstream/master'

**Comment by LazennG** [comment]
ALMOST ULTRAKILL LETS GOOOOOOOOO

**Review by nesisknesis — APPROVED** [review]
